client/components/platform-checkout/save-user/additional-information.js

```diff
@@ -17,11 +18,24 @@ const AdditionalInformation = () => {
 					<PhoneIcon />
 				</div>
 				<span>
-					{ __(
-						'Enter your mobile phone number to save your checkout information for faster checkouts here, ' +
-							'and at other stores powered by WooPay.',
-						'woocommerce-payments'
-					) }
+					{ interpolateComponents( {
+						mixedString: __(
+							'Enter your mobile phone number to save your checkout information for faster checkouts here, ' +
+								'and at other stores powered by {{wooPayLink/}} .',
```

Comment:
Looks like there's an extra space between the end of `{{wooPayLink/}}` and the `.`

Reply:
Thank you for the feedback, @brianyu28!